### ISSUE TITLE
[Enhancement] reduce log spam from the batch-module

### ISSUE THREAD

**Issue by InteractiveNinja** [bbottema/simple-java-mail#543]
Hello 👋 

Firstly, thanks for maintaining this library.

I noticed that when I upgraded to >=v8.11.1, the logs are getting flooded with messages when I run the backend server locally.
I found out that the log statement is from https://github.com/bbottema/generic-object-pool/pull/8, which is used in https://github.com/bbottema/simple-java-mail/issues/529.
```
15:44:01.147 [pool-1-thread-2|org.bbottema.genericobjectpool.GenericObjectPool]                                                       DEBUG - 0 objects invalidated as per expiration policy!
15:44:01.157 [pool-1-thread-2|org.bbottema.genericobjectpool.GenericObjectPool]                                                       DEBUG - 0 objects invalidated as per expiration policy!
15:44:01.167 [pool-1-thread-2|org.bbottema.genericobjectpool.GenericObjectPool]                                                       DEBUG - 0 objects invalidated as per expiration policy!
15:44:01.177 [pool-1-thread-2|org.bbottema.genericobjectpool.GenericObjectPool]                                                       DEBUG - 0 objects invalidated as per expiration policy!
15:44:01.188 [pool-1-thread-2|org.bbottema.genericobjectpool.GenericObjectPool]                                                       DEBUG - 0 objects invalidated as per expiration policy!
15:44:01.198 [pool-1-thread-2|org.bbottema.genericobjectpool.GenericObjectPool]                                                       DEBUG - 0 objects invalidated as per expiration policy!
```
I don't think there's anything I need to change to stop the log statements without removing the logging for this library in my project.

I appreciate your time 👍

**Comment by bbottema** [comment]
Well, I think it's good to tweak it so it only logs if there are actual objects invalidated and also perhaps make it TRACE level. Having said that, wouldn't it make sense for you to set the debug level to INFO for `org.bbottema.genericobjectpool`?

**Comment by bbottema** [comment]
Released tweak in 8.11.3

**Comment by InteractiveNinja** [comment]
Thank you kindly for the fast change 👍
